### PR TITLE
[Cleanup]: Fix Android Studio Warnings #13282

### PR DESCRIPTION
**Pull request by isbvah**
## Purpose / Description
_Describe the problem or feature and motivation_

## Fixes
* #13282

## Approach
fixed by basic conventions first I scanned the warnings and then fixed some of the warnings under kotlin hood

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt [hunk 1]
```diff
@@ -610,6 +610,7 @@ abstract class AbstractFlashcardViewer :
         }
     }
 
+    @Deprecated("This function is deprecated due to ...", ReplaceWith("..."))
```
Comment (neeldoshii) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt:613]: Not sure what this statement means `deprecated due to ..."` instead you can use `@Deprecated("Deprecated in Java")`

**Review by neeldoshii — CHANGES_REQUESTED** [review]

**Comment by neeldoshii** [comment]
Thank you for contributing. LGTM, left one minor review.

**Comment by isbvah** [comment]
Done Minor Fix

**Comment by david-allison** [comment]
@isbvah, your git commit email means that the commits here are attributed to https://github.com/ishqDehlvi

Is this your intention?

File: AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt [hunk 2]
```diff
@@ -1538,9 +1539,9 @@ open class CardBrowser :
 
     /**
      * Removes cards from view. Doesn't delete them in model (database).
-     * @param reorderCards Whether to rearrange the positions of checked items (DEFECT: Currently deselects all)
+     * @param this@removeNotesView Whether to rearrange the positions of checked items (DEFECT: Currently deselects all)
      */
-    private fun removeNotesView(cardsIds: List<Long>, reorderCards: Boolean) {
+    private fun Boolean.removeNotesView(cardsIds: List<Long>) {
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt:1544]: Revert changes to this method, we want to keep it for now

File: AnkiDroid/src/main/java/com/ichi2/ui/AppCompatPreferenceActivity.kt [hunk 3]
```diff
@@ -67,8 +67,13 @@ abstract class AppCompatPreferenceActivity<PreferenceHack : AppCompatPreferenceA
         val summaries: MutableMap<String, String?> = HashMap()
         protected val listeners: MutableList<SharedPreferences.OnSharedPreferenceChangeListener> = LinkedList()
 
-        @KotlinCleanup("scope function")
-        abstract fun cacheValues()
+        init {
+            this.cacheValues()
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/ui/AppCompatPreferenceActivity.kt:71]: revert this file and just remove the `KotlinCleanup`

**Review by david-allison — CHANGES_REQUESTED** [review]
Looks great! A couple of nitpicks, then let's get this in

**Review by david-allison — APPROVED** [review]

**Pull request merged**